sdk/loadtestservice/azure-developer-loadtesting/azure/developer/loadtesting/aio/operations/_operations.py

```diff
@@ -437,301 +1154,356 @@ async def get_app_component_by_name(self, name: str, **kwargs: Any) -> JSON:
-                            "name": {
-                                "localizedValue": "str",  # Metric localized name.
-                                  Required.
-                                "value": "str"  # Metric name value. Required.
-                            },
-                            "resourceId": "str",  # Azure resource Id. Required.
-                            "resourceType": "str",  # Azure resource type. Required.
-                            "displayDescription": "str",  # Optional. Metric description.
-                            "id": "str",  # Optional. Unique identifier for metric.
-                            "unit": "str"  # Optional. Metric unit.
-                        }
-                    },
-                    "name": "str",  # Optional. Server metrics config name.
-                    "testId": "str",  # Optional. [Required, if testRunId is not given] Load test
-                      unique identifier.
-                    "testRunId": "str"  # Optional. [Required, if testId is not given] Load test
-                      run unique identifier.
-                }
+            return pipeline_response
 
-                # response body for status code(s): 200, 201
-                response == {
-                    "metrics": {
-                        "str": {
-                            "aggregation": "str",  # Metric aggregation. Required.
-                            "metricnamespace": "str",  # Metric name space. Required.
-                            "name": {
-                                "localizedValue": "str",  # Metric localized name.
-                                  Required.
-                                "value": "str"  # Metric name value. Required.
-                            },
-                            "resourceId": "str",  # Azure resource Id. Required.
-                            "resourceType": "str",  # Azure resource type. Required.
-                            "displayDescription": "str",  # Optional. Metric description.
-                            "id": "str",  # Optional. Unique identifier for metric.
-                            "unit": "str"  # Optional. Metric unit.
-                        }
-                    },
-                    "name": "str",  # Optional. Server metrics config name.
-                    "testId": "str",  # Optional. [Required, if testRunId is not given] Load test
-                      unique identifier.
-                    "testRunId": "str"  # Optional. [Required, if testId is not given] Load test
-                      run unique identifier.
-                }
-        """
+        return AsyncItemPaged(get_next, extract_data)
 
-    @overload
-    async def create_or_update_server_metrics_config(
-        self, name: str, body: IO, *, content_type: str = "application/merge-patch+json", **kwargs: Any
+    @distributed_trace_async
+    async def upload_test_file(
```

Comment:
can you run the samples? You have patched this operation as `upload_test_file`, but you sample is running `begin_upload_test_file`. The first is a regular operation, and the sample is an LRO operation. Which one is it supposed to be?